prefix.py

```diff
@@ -71,37 +71,63 @@
 When generating a map, you must include code that calls the initMap() function to display the map. This function should be included in the generated JavaScript code.
 
-When asked to generate a pdf report, provide the code necessary to produce the pdf report with python and reportlab. Do not include any additional text, explanations, or descriptions.
+When asked to generate a PDF report, ensure the following design improvements for visual appeal:
+
+1. **Fonts**:
+   - Use a modern font like Helvetica or Times New Roman for a clean look.
+   - Headers should be bold and at least 16pt, while body text can be 12pt.
+   - Use italic or bold styles for emphasis where needed.
+
+2. **Color Scheme**:
+   - Apply soft background colors to sections (e.g., light grey or pale blue) for separation and readability.
+   - Table headers should have a distinct color, such as dark blue or grey, with white text for contrast.
+
+3. **Tables**:
+   - Add alternating row colors to improve readability, such as alternating between white and light grey.
+   - Use borders around cells and add padding for a clean, structured appearance.
+   - Use a larger font for headers to make them stand out.
+
+4. **Page Layout**:
+   - Ensure ample white space for readability by adding margins and spacing between sections.
+   - Add a title at the top of the page in a larger font (e.g., 20pt) and centered for emphasis.
+   - Include page numbers at the bottom if the report spans multiple pages.
+
+5. **Images and Logos**:
+   - If requested, add the company logo or branding at the top of the report.
+   - Ensure logos are properly aligned and sized to not overlap with text.
+
+6. **Charts and Visuals**:
+   - If the report includes charts, use clear labeling and legends.
+   - Embed the chart directly in the report with consistent styling (fonts, colors, and borders).
 
-For example, use the following structure for creating a marker:
+By following these steps, your reports will be visually appealing, professional, and easy to read. Always focus on clean, structured design principles.
 
 {marker_boilerplate}
 
-If a prompt asks for location data which doesn't pertain to buildings, then use the "google_places" tool to find the relevant information.
-When finding distance relative to a building or when prompted for the closest or nearest location to the building, use google_maps_directions with the lat and lon of the core_condobuilding table (ALWAYS WITH THE lat and lon) and the destination address to calculate the distance between the building the destination, repeat for all available destinations and return the destination with the lowest distance from the building.
+If a prompt asks for location data which doesn't pertain to buildings, then use the Google Places tool to find the relevant information.
```

Comment:
Capitalization and spelling were not part of this issue, just fix the reports and leave the rest alone as it was

Reply:
okay sure thing